plugins/techdocs/src/home/components/OwnedContent.tsx

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { useCopyToClipboard } from 'react-use';
+import { generatePath } from 'react-router-dom';
+
+import { IconButton, Tooltip } from '@material-ui/core';
+import ShareIcon from '@material-ui/icons/Share';
+import {
+  Content,
+  ContentHeader,
+  SupportButton,
+  Table,
+  identityApiRef,
+  useApi,
+  EmptyState,
+  Button,
+  SubvalueCell,
+  Link,
+} from '@backstage/core';
+import { Entity } from '@backstage/catalog-model';
+
+import { rootDocsRouteRef } from '../../plugin';
+
+export const OwnedContent = ({ value }: { value: Entity[] | undefined }) => {
+  const [, copyToClipboard] = useCopyToClipboard();
+  const identityApi = useApi(identityApiRef);
+  const userId = identityApi.getUserId();
+
+  if (!value || !userId) return null;
+
+  const ownedDocuments = value
+    .filter((entity: Entity) => entity?.spec?.owner === userId)
+    .map(entity => {
+      return {
+        name: entity.metadata.name,
+        description: entity.metadata.description,
+        owner: entity?.spec?.owner,
+        type: entity?.spec?.type,
```

Comment:
Not directly related to this but just a general idea / thought.  Now that systems and domains are supported in the catalog, maybe it would be worth opening up docs support to more then just components. I suppose the difference being the actual docs site would be a component, but it could be associated with a different entity kind via an annotations maybe. Anyways not in scope for this PR, just thinking.

Reply:
Hmm yes agree, great idea, I was just thinking "do we have any restrictions on that `kind` needs to be Component for TechDocs"? I guess we do support more then that 🤔  But I might be wrong, @OrkoHunter maybe can give us the latest update here!

Reply:
Hmm, I don't think there is any restriction to `kind` being `Component` for TechDocs and it could be anything, as long as there is `backstage.io/techdocs-ref` annotation in the entity.

Reply:
Haven't kept up with the development, but is there a possibility of conflicting behaviour if the same techdocs annotation is used in multiple entities. For example it's attached to the component of type documentation for the repo that provides the docs and a domain or system that they are related to.

Reply:
There shouldn't be any conflicts if two Entities have same annotation. The generated sites are stored in and retried from `:namespace/:kind/:name/` directory in the storage. So, same techdocs-ref annotation for multiple entities only means that the source code is common, but the sites are going to be unique, since both are different entities. Hope this makes sense.

Reply:
Perfect. That makes this idea already supported :)

Reply:
@andrewthauer The thing I can think of now is of course that if you do have a system-kind entity pointing to a techdocs ref for documentation and then you have a domain-kind entity pointing to the same techdocs ref as you would like these entities to point to the same documentation. That will cause publishing of two different docs sites, where the documentation is the same, right? @OrkoHunter So that one could see as kind of a conflict...

Reply:
Yeah, sites will still be duplicated, but I think it's best that they are unique per entity. :)

Reply:
So we have this scenario now. We have domains and we also some domains have a dedicated docs-as-code repo for documentation related to the domain. Currently the docs only repo is registered as a component but it would be nice to natively associate it with its applicable domain. Are you saying we should register the repo with the domain entity instead of the component?

Reply:
> Currently the docs only repo is registered as a component
> Are you saying we should register the repo with the domain entity instead of the component?

Yes! You shouldn't have to create a new Component just to view docs of a Domain.

Assuming you are on the latest techdocs and techdocs-backend plugins and using techdocs-cli on CI/CD ->

Publish the docs on CI with the CLI option `--entity` to be `:namespace/:kind/:name` such that `:kind` is `System` https://github.com/backstage/techdocs-cli#publish-generated-techdocs-sites

Add `backstage.io/techdocs-ref: url:TARGET` annotation to your domain's `catalog-info.yaml` to enable TechDocs site for it in Backstage frontend. And when you open the TechDocs site for it, the site will be fetched from the storage for the domain using the unique `:namespace/:kind/:name` in the URL of the site.

I should probably investigate more on Systems and Domains https://backstage.io/docs/features/software-catalog/system-model since they represent a collection of Entities but for TechDocs, they are not that different from an entity like Component as long as the triplet `:namespace/:kind/:name` is unique and entity can have a `backstage.io/techdocs-ref` metadata annotation.